network/ServerNetworking.h

```diff
@@ -227,8 +227,9 @@ class PlayerConnection :
     const std::string& ClientVersionString() const;
 
     /** Returns true iff the client's specified version string matches the
-      * version string of this server process. */
-    bool ClientVersionStringMatchesThisServer() const;
+      * version string of this server process so the server will enable binary serialization if it
+      * is allowed. */
```

Comment:
Comment now misleading; should / could just note that it indicates whether the server will use binary serialization, not how it determines this.